### PR TITLE
FIX: final workflow changes

### PR DESCRIPTION
**Pull request by eilidhmacnicol**

### CODE REVIEW

File: fmriprep_rodents/workflows/base.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def init_single_subject_wf(subject_id):
             f"task <{task_id or 'all'}>. All workflows require BOLD images."
         )
 
-    workflow = Workflow(name=f"single_subject_{subject_id}_wf")
+    workflow = Workflow(name=name)
```
Comment (oesteban) [inline fmriprep_rodents/workflows/base.py:150]: You'll need to revert this one
```suggestion
    workflow = Workflow(name=f"single_subject_{subject_id}_wf")
```

**Review by oesteban — APPROVED** [review]
Looking great!

**Review by oesteban — APPROVED** [review]
Great stuff!

EDIT: this is a GH glitch

File: fmriprep_rodents/workflows/base.py (hunk 1 above)
Comment (eilidhmacnicol) [reply fmriprep_rodents/workflows/base.py:150]: Oh - in a versionI was looking at, `name` was defined on line 132. Happy to oblige 👍🏼

**Pull request merged**